less/mixins.less

```diff
@@ -1 +1,83 @@
 // Mixins below variables
+
+.width-start-mixin() {
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+ 
+}
+
+.width-start-mixin-footer() {
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+}
+
```

Comment:
So these `.width-start-mixin()` and `.width-start-mixin-footer()` Are the exact same styles so you don't need two separate mixins for it.

Remember mixins are just for grouping styles together and being able to pass that from place to place.

These two _might_ not be necessary since I think these styles will be inherited anyway. I could be wrong lol